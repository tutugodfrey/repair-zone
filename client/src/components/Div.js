import React, {Component} from 'react';

export default class Div extends Component {
  render() {
    return  (
      <div id = {this.props.divId} className= {this.props.divClass}>
      {this.props.content}
      </div>
    )
  }
}