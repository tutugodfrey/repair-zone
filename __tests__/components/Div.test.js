import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

// Component to be tested
import Div from '../../client/src/components/elementComponents/Div';
describe('<Div />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Div />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});