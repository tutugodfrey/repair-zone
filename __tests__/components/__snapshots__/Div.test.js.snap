// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Div /> render() renders the component 1`] = `<div />`;
